spec/features/courses/subjects/edit_spec.rb

```diff
@@ -52,7 +54,7 @@
 
       subjects_page.master_subject_fields.select(subjects.first.subject_name)
       subjects_page.save.click
-      expect(languages_page).to be_displayed
+      expect(current_url).to eql("http://www.example.com/organisations/#{provider.provider_code}/2020/courses/#{course.course_code}/details")
```

Comment:
To be consistent with the other tests within publish we should expect the `details_page` page object to be displayed (not sure what the class is for it off the top of my head)

Reply:
The reason we aren't doing this is because site prism is notoriously inconsistent with handling URL parameters so like we're doing on Find, we should just entirely avoid it.